src/transeq.f90

```diff
@@ -764,10 +764,8 @@ subroutine scalar_transport_eq(dphi1, rho1, ux1, uy1, uz1, phi1, schmidt, is_eve
       tb1(:,:,:) = ux1(:,:,:) * tb1(:,:,:)
     endif
     if (skewsc) then
-      ! (1 - 0.5) * ux.dTdx + 0.5 * (d(ux.T)dx - T.duxdx)
+      ! (1 - 0.5) * ux.dTdx + 0.5 * d(ux.T)dx
```

Comment:
should be (one-half) and half

Reply:
This is fixed in the updated version of this PR